images/linux/scripts/helpers/install.sh

```diff
@@ -50,6 +54,19 @@ verlte() {
     [  "$1" = "$sortedVersion" ]
 }
 
+verify_http_status_code()
+{
+    URL="$1"
+    http_code=$(curl -sL -o out.html -w '%{http_code}' $URL)
+
+    if [[ $http_code -eq 200 ]]; then
+        return 0
+    else
+        echo "Bad HTTP status code: $http_code for the provided link: $URL."
+        return 1
+    fi
```

Comment:
Looks like this block can be simplified
```suggestion
    if [[ $http_code -ne 200 ]]; then
        echo "Bad HTTP status code: $http_code for the provided link: $URL."
        return 1
    fi
```